src/Windows/Avalonia.Win32/SystemDialogImpl.cs

```diff
@@ -67,6 +67,16 @@ internal class SystemDialogImpl : ISystemDialogImpl
 
                     frm.SetFileTypeIndex(0);
 
+                    if (dialog is SaveFileDialog saveFileDialog)
+                    {
+                        var overwritePrompt = saveFileDialog.ShowOverwritePrompt ?? true;
+
+                        if (!overwritePrompt)
+                        {
+                            frm.SetOptions(frm.Options &~ FILEOPENDIALOGOPTIONS.FOS_OVERWRITEPROMPT);
```

Comment:
It's better to move where the rest of option calculation happens - https://github.com/AvaloniaUI/Avalonia/blob/afc00162984007c345a6731802fabfdb56842c80/src/Windows/Avalonia.Win32/SystemDialogImpl.cs#L39

Reply:
Addressed